addons/packages/kapp-controller/0.30.1/bundle/.imgpkg/bundle.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: imgpkg.carvel.dev/v1alpha1
+kind: Bundle
+metadata:
+  name: kapp-controller
+authors:
+- name: Daniel Guo
```

Comment:
Here the name and email should be yours as you are making the change

Reply:
Added my name